packages/react-components/src/Concepts/Positioning/PositioningDefault.stories.tsx

```diff
@@ -0,0 +1,7 @@
+import * as React from 'react';
+import { PositioningProps } from '@fluentui/react-positioning';
+import { PositionedComponent } from './utils.stories';
+
+export const Default = (props: PositioningProps) => {
+  return <PositionedComponent positioning={props} />;
```

Comment:
`align` does not do anything unless position is also set. Do we want to document it somehow?